packages/react/src/Column/Column.tsx

```diff
@@ -20,8 +20,8 @@ export type ColumnProps = {
 } & PropsWithChildren<HTMLAttributes<HTMLElement>>
 
 export const Column = forwardRef(
-  ({ as: Tag = 'div', children, className, gap = 'medium', ...restProps }: ColumnProps, ref: any) => (
-    <Tag {...restProps} ref={ref} className={clsx('ams-column', `ams-column--gap-${gap}`, className)}>
+  ({ as: Tag = 'div', children, className, gap, ...restProps }: ColumnProps, ref: any) => (
+    <Tag {...restProps} ref={ref} className={clsx('ams-column', gap && `ams-column--gap-${gap}`, className)}>
```

Comment:
This is correct and the "no" gap should return false here.

Reply:
Only if we don’t merge this PR. The idea is that `Row` and `Column`, just like `Grid` render a gap between their children by default.

We add the ‘none’ option to `Row` and `Column` to support (edge) cases where that is unwanted. We could even wait to add that option until people need it.